src/components/input/radio/button/style.css

```diff
@@ -0,0 +1,27 @@
+:host {
+	display: flex;
+	margin-right: 1em;
+	align-items: center;
+	position: relative;
+}
+
+:host:hover > * {
+	cursor: pointer;
+}
+
+:host input[type="radio"]:focus-visible + smoothly-icon {
+	outline: 3px solid rgb(var(--smoothly-primary-color));	
```

Comment:
This is an ongoing discussion we have about how we want to use colors. 

Using smoothly-primary-color makes removes the ability of using the color prop to change the color the component uses. Instead use smoothly-color and set the color property to primary. 

Now, that is going to open up another can of worms when it comes to styling the component so you might want to ignore this comment and just leave this.